### ISSUE TITLE
Tighter integration with the Web Audio API

### ISSUE THREAD

**Issue by oampo** [oampo/Audiolet#9]
This ability to offload some of the processing to C++ would be cool, and would add a fair chunk of DSP code without having to do too much work.  The difficult part of this is moving from variable length buffers to fixed length buffers in the graph, and any overheads associated with moving between the C++ and the JS.

**Comment by forresto** [comment]
:+1:

**Comment by forresto** [comment]
Now that Web Audio API is getting implemented in Chrome and iOS 6, are there some more thoughts on this?

**Comment by oampo** [comment]
@forresto Yeah, I've been thinking about this quite a lot recently - I wrote a little bit about how I imagine this will happen at h5bp/lazyweb-requests#82.  I want to have a go at migrating the core of Audiolet to the Web Audio API proper when I get a moment.  The only reservation I have is that there are loads of changes and discussions around the API taking place at the moment, and I don't want to fully jump into this while the API is liable to change drastically.

**Comment by micahscopes** [comment]
Any word on this?  I'd like to use Audiolet but sink.js doesn't seem to work well (at all) with Firefox 28
